.circleci/config.yml

```diff
@@ -2,8 +2,22 @@
 version: 2.1
 
 orbs:
-  architect: giantswarm/architect@4.3.0
+  architect: giantswarm/architect@4.4.0
 
+commands:
+  run-ats:
+    parameters:
+      ats_version:
+        type: string
+        default: 0.1.4
+      chart_archive_prefix:
+        type: string
+    steps:
+      - architect/run-tests-with-ats:
+          chart_archive_prefix: "<< parameters.chart_archive_prefix >>"
+          app-test-suite_version: "v<< parameters.ats_version >>"
+          app-test-suite_container_tag: "<< parameters.ats_version >>"
```

Comment:
I'm a bit triggered by the hypens and underscores here :) but it makes sense.

Same for ordering the parameters but there aren't that many.

Reply:
I think we use underscores all over the place in architect-orb as well. Not really sure why :sweat_smile: